src/server/auth/server/roles.go

```diff
@@ -185,6 +223,10 @@ func init() {
 			licenseAdminRole.Permissions,
 			secretAdminRole.Permissions,
 			pachdLogReaderRole.Permissions,
+			projectViewer.Permissions,
```

Comment:
you shouldn't need `projectViewer.Permissions, projectWriter.Permissions` here because they are implied by `projectOwner.Permissions`